docs/src/main/paradox/slick.md

```diff
@@ -1,6 +1,6 @@
 # Slick (JDBC)
 
-The Slick connector provides Scala and Java DSLs to create a `Source` to stream the results of a SQL database query and a `Flow`/`Sink` to perform SQL actions (like inserts, updates, and deletes) for each element in a stream. It is built on the [Slick](http://slick.lightbend.com/) library to interact with a long list of [supported relational databases](http://slick.lightbend.com/doc/$slickVersion$/supported-databases.html).
+The Slick connector provides Scala and Java DSLs to create a `Source` to stream the results of a SQL database query and a `Flow`/`Sink` to perform SQL actions (like inserts, updates, and deletes) for each element in a stream. It is built on the [Slick](http://slick.lightbend.com/) library to interact with a long list of @extref[supported relational databases](slick:supported-databases.html).
```

Comment:
But the links did work, this is shorter though.

Reply:
It was broken after https://github.com/akka/alpakka/pull/1015 and can be seen here: https://developer.lightbend.com/docs/alpakka/snapshot/slick.html